TreeDB/collections/document_ingestion_test.go

```
@@ -967,20 +1030,94 @@ func TestIngestSourcesCommitAmbiguityBoundariesReopenAndRetry(t *testing.T) {
+			if err != nil {
+				t.Fatalf("OpenCollection after cut: %v", err)
+			}
+			parentRaw, err := reopenedCol.Get([]byte("src-0"))
+			if err != nil {
+				t.Fatalf("Get parent after cut: %v", err)
+			}
+			var parent map[string]any
+			if err := json.Unmarshal(parentRaw, &parent); err != nil {
+				t.Fatalf("decode parent after cut: %v", err)
+			}
+			if gotNew := strings.Contains(parent["body"].(string), "tok7"); gotNew != tc.wantNew {
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)